packages/modal/package.json

```diff
@@ -14,9 +14,11 @@
   "dependencies": {
     "@leafygreen-ui/hooks": "^1.1.0",
     "@leafygreen-ui/icon": "^2.1.0",
+    "@leafygreen-ui/icon-button": "2.0.0",
     "@leafygreen-ui/lib": "^3.2.0",
     "@leafygreen-ui/palette": "^1.1.1",
     "@leafygreen-ui/portal": "^1.1.8",
-    "react-transition-group": "^4.2.1"
+    "react-transition-group": "^4.2.1", 
+    "react-focus-lock": "^2.2.1"
```

Comment:
I don't think it's a good idea to use this library:

- It doesn't seem like it's maintained anymore, last update was over a year ago.
- It hasn't been updated to the latest React APIs, meaning it'll have deprecation warnings in the console.

I'd be more comfortable to port it into leafygreen (either as a child inside modal or as its own package) and update it to reflect current best React practices. This should also include tests for this.